src/views/List.vue

```diff
@@ -104,8 +117,7 @@ export default {
     },
   },
   mounted() {
-    const path = this.$route.params.slug.split('/');
-    this.fetchList({ slug: path[path.length - 1] });
+    this.fetchList({ slug: this.slug });
```

Comment:
we should actually do this before the route is resolved. we can use `beforeRouteUpdate` and `beforeRouteEnter`.